output-tests/README.md

```diff
@@ -0,0 +1,26 @@
+These tests are intended to validate that implementations are correctly generating output in accordance with the specification.
+
+Output was initially specified with draft 2019-09.  It remained unchanged for draft 2020-12 <!-- do we need explicit tests for 2002-12? -->, but will receive an update with the next release.
+
+## Organization
+
+The tests are organized into two categories: content and structure.
+
+Content tests verify that the keywords are producing the correct annotations.  Since there are no requirements on the content of error messages, there's not much that can be verified for them, if they're even generated.  These tests need to extensively cover the annotation behaviors of each keyword.  The expected output format for these tests is `basic`.
```

Comment:
Not much sense bikeshedding at this point so feel free to ignore, but should we consider using the "existing terminology" here and call these `annotation tests`? That's what they seem to cover, right?

Reply:
Maybe I need to return this.  There _are_ some tests that make sure that an error is the right type if present.  I'm not sure if that's useful, but we can discuss.